packages/react-openapi/src/OpenAPICodeSample.tsx

```diff
@@ -19,6 +19,19 @@ export function OpenAPICodeSample(props: {
 }) {
     const { data, context } = props;
 
+    const requiredHeaders = data.operation.parameters
+        ?.map(noReference)
+        .filter((param) => param.in === 'header' && param.required);
+
+    const headersObject: { [k: string]: string } = {};
+    requiredHeaders?.forEach((header) => {
+        const example = header.schema && generateSchemaExample(noReference(header.schema));
+        if (example) {
```

Comment:
You should be careful with `&&` I see you use it a lot and you sometimes miss what it means when the function can also return something negative.

```suggestion
        const example = header.schema ? generateSchemaExample(noReference(header.schema)) : undefined;
        if (example !== undefined) {
```

`generateSchemaExample` can return `null` | `false`, etc

Reply:
Thank you @SamyPesse! Noted! Won't happen again.